Lib/test/test_compileall.py

```diff
@@ -68,6 +69,7 @@ def setUp(self):
 
     def tearDown(self):
         shutil.rmtree(self.directory)
+        multiprocessing_cleanup_tests()
```

Comment:
Here, you could presumably call `addCleanup()` in `setUp()`?

Reply:
Ok, done.